### PR TITLE
footer works and is always at the bottom no matter position in scroll…

### PR DESCRIPTION
**Pull request by jstavh**
Created the footer component. The footer always appears at the bottom no matter the position when scrolling or screen size.

### CODE REVIEW

File: src/astro/src/layouts/CareersLayout.astro [hunk 1]
```diff
@@ -32,4 +32,7 @@ const url = Astro.url;
     {url.pathname !== "/careers" && <Join url={url} />}     
   </article>  
 </body>
+<footer>
```
Comment (robinbb) [inline src/astro/src/layouts/CareersLayout.astro:25]: Footer element is doubly wrapped.

File: src/astro/src/layouts/index.astro [hunk 2]
```diff
@@ -14,4 +15,7 @@ const {title} = Astro.props;
   <Nav />
   <Front /> 
 </body>
+<footer>
```
Comment (robinbb) [inline src/astro/src/layouts/index.astro:16]: Doubly wrapped footer element.

**Review by robinbb — CHANGES_REQUESTED** [review]

File: src/astro/src/layouts/CareersLayout.astro (hunk 1 above)
Comment (jstavh) [reply src/astro/src/layouts/CareersLayout.astro:25]: I removed from the component so now only wrapped once.

File: src/astro/src/layouts/index.astro (hunk 2 above)
Comment (jstavh) [reply src/astro/src/layouts/index.astro:16]: I removed from the component so now only wrapped once.

**Comment by robinbb** [comment]
Copyright statement does not appear on home page.

**Comment by jstavh** [comment]
> Copyright statement does not appear on home page.

Tiny change to make sure it works on all screen sizes.

**Review by robinbb — APPROVED** [review]

**Pull request merged**